src/codeGenerators/orchestration/nodejs/toOrchestration.ts

```diff
@@ -209,7 +213,9 @@ export default function codeGenerator(node: any, options: any = {}): any {
       if (options?.lhs) return `${node.name}.${node.memberName}`;
       return codeGenerator({ nodeType: 'Identifier', name: `${node.name}.${node.memberName}`, subType: node.subType });
 
-
+    case 'IndexAccess': 
```

Comment:
I don't think we need an `IndexAccess` here - doesn't cause issues but by the code gen they should have been edited to be commitment/variableDec/identifier types :)